README.md

```diff
@@ -64,8 +64,8 @@ Coming soon...
 Using domJSON is super simple: use the [`.toJSON()`](#domJSON.toJSON) method to create a JSON representation of the DOM tree:
 
 ```javascript
-var someDOMElement = document.getElementById('sampleId');
-var jsonOutput = domJSON.toJSON(myDiv);
+let someDOMElement = document.getElementById('sampleId');
+let jsonOutput = domJSON.toJSON(someDOMElement);
```

Comment:
The main change of the PR is just to change `myDiv` to `someDOMElement`.

Also, changed `var` to `let` to encourage "immutability by default". Perhaps too preachy??